Makefile

```diff
@@ -2,7 +2,7 @@
 
 build:
 	if [ ! -d "runtime" ]; then \
-		mkdir runtime/configs -p; \
+		mkdir -p runtime/configs; \
```

Comment:
Update arguments order according to the specifications: https://pubs.opengroup.org/onlinepubs/7990989799/xcu/mkdir.html